config/routes.rb

```diff
@@ -1,11 +1,18 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :admin do
+      get 'users/show'
+    end
+  end
+  get 'users/show'
```

Comment:
This route needs to go. But I'm a bit confused, why do you have 2 namespaces with the same name?

Reply:
Unnoticed addition generated by rails. Removed.

Reply:
I still see the extra route